cpp/tests/utilities/column_wrapper.cuh

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef COLUMN_WRAPPER_H
```

Comment:
I think we should call all of our C++ headers `.hpp` and all of our C headers `.h`

Reply:
This is a header the includes device code (thrust). As such, it uses the `cuh` header extension.